config/config.exs

```diff
@@ -32,3 +32,9 @@ config :phoenix, :format_encoders,
 config :mime, :types, %{
   "application/vnd.api+json" => ["json-api"]
 }
+
+config :guardian, Guardian,
+  issuer: "MyApp",
+  ttl: { 1, :hours },
+  secret_key: %{"k" => "X5uaThfmNGtgdwaYJJjUFA", "kty" => "oct"},
```

Comment:
Nope. We overwrite this config in a secret production file that doesn't get checked in to version control. This is only for dev and testing.